.github/workflows/ci.yml

```diff
@@ -166,79 +174,73 @@ jobs:
             tar xvf "$file"
           done
 
-      - uses: actions/setup-java@v3
+      - name: Setup Java
+        uses: actions/setup-java@v4
         with:
-          distribution: 'zulu'
-          java-version: 17
-      - name: Fetch the Ghidra dependencies.
-        uses: eskatos/gradle-command-action@v2
+          distribution: temurin
```

Comment:
Why change from zulu to temurin?

Reply:
`temurin` is cached in the build image and doesn't need to be downloaded, which seemed nice, but in reality it's only a 5s download and extract, so I'm happy to revert 🙂

Reply:
That does seem nice. I googled, and it looks like `temurin` is handled with the Eclipse Foundation, which I guess should be more future-proof too. Let's keep this.